osu.Game/Database/BeatmapDatabase.cs

```diff
@@ -144,7 +144,11 @@ public void Import(ArchiveReader archiveReader)
         public void Import(params string[] paths)
         {
             foreach (string p in paths)
-                Import(p);
+            {
+                //Somehow this fixes a crash when importing a beatmap twice in quick succession

```

Comment:
I was confused because I thought it would delete it immediately and so you wouldn't be able to import it again. But I guess it's because osu schedules the import as a task so it only happens a little later.